python/requirements/ml/requirements_train.txt

```diff
@@ -9,9 +9,17 @@ tensorboardX==2.4.1
 # Dependencies for Hugging Face examples & tests:
 # `python/ray/train/examples/transformers/transformers_example.py`
 # `python/ray/air/examples/huggingface/huggingface_basic_language_modeling_example.py`
+# `python/ray/train/tests/test_accelerate_trainer_gpu.py`
 # `python/ray/air/tests/test_huggingface_trainer.py`
+
+# We need transformers>=4.19.0 for HuggingFaceTrainer to work (not available on py 3.6)
+# If changing the version here, also change it in HuggingFaceTrainer docstring
 transformers==4.18.0; python_version <= '3.6'
 transformers==4.19.1; python_version > '3.6'
-accelerate==0.5.1
+# We need accelerate>=0.17.0 for AccelerateTrainer to work (not available on py 3.6)
+# If changing the version here, also change it in AccelerateTrainer docstring
+accelerate==0.5.1; python_version <= '3.6'
+accelerate==0.17.1; python_version > '3.6'
+deepspeed; python_version > '3.6'
```

Comment:
hmm, now deepspeed will always get installed regardless of whether users actually need it or not?